islandora.module

```diff
@@ -316,6 +316,43 @@ function islandora_preprocess_node(&$variables) {
   }
 }
 
+/**
+ * Implements hook_form_alter().
+ */
+function islandora_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $media_add_forms = ['media_audio_add_form', 'media_document_add_form',
+    'media_extracted_text_add_form', 'media_file_add_form', 'media_image_add_form',
+    'media_fits_technical_metadata_add_form', 'media_video_add_form',
+  ];
+
+  if (in_array($form['#form_id'], $media_add_forms)) {
+    $params = \Drupal::request()->query->all();
+    $media_of_nid = $params['edit']['field_media_of']['widget'][0]['target_id'];
+    $node = \Drupal::entityTypeManager()->getStorage('node')->load($media_of_nid);
+    $form['name']['widget'][0]['value']['#default_value'] = $node->getTitle();
```

Comment:
Adding a media without a `node` originally associated whitescreens in the same way as below.

Reply:
OK, thanks, I will investigate.

Reply:
@jordandukart OK, the two WSODs you found are gone, and the watchdog is clean. Sorry about the sloppiness there and for wasting your testing time.

Reply:
No worries!